camunda-formio-tasklist-vue/src/components/FormListModal.vue

```diff
@@ -124,12 +133,13 @@ export default class FormListModal extends Mixins(BaseMixin) {
   private formperPage: number = 10;
   private formNumPages: number = 1;
   private formcurrentPage: number = 1;
-  private formValueId: string | null = null;
-  private formId: string | null = null;
-  private submissionId: string | null = null;
-  private formioUrl: string | null = null;
+  private formValueId?: string  = "";
+  private formId?: string = undefined;
+  private formioUrl?: string = undefined;
   private formTitle: string = "";
   private formsflowAIApiUrl: any;
```

Comment:
pls avoid any, you can add string for url

Reply:
Thanks for spotting it :)